pkg/master/master_test.go

```diff
@@ -681,7 +737,7 @@ func testInstallThirdPartyAPIPostForVersion(t *testing.T, version string) {
 		},
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Foo",
-			APIVersion: version,
+			APIVersion: "company.com/" + version,
```

Comment:
@brendandburns, after this PR, the config file of a thirdparty resource needs to be filled with "group/version" in the APIVersion field. This behavior is consistent with other groups. Please let me know if you think this is not proper.